.nuspec/Microsoft.Maui.Resizetizer.targets

```
@@ -248,13 +248,13 @@
 
     <Target Name="ProcessMauiAssets">
         <ItemGroup Condition="'$(_ResizetizerIsAndroidApp)' == 'True'">
-            <AndroidAsset Include="@(MauiAsset)" Link="%(FileName)%(Extension)" />
+            <AndroidAsset Include="@(MauiAsset)" Link="%(RecursiveDir)\%(FileName)%(Extension)" />
         </ItemGroup>
         <ItemGroup Condition="'$(_ResizetizerIsiOSApp)' == 'True'">
-            <Content Include="@(MauiAsset)" Link="%(FileName)%(Extension)" />
+            <Content Include="@(MauiAsset)" Link="%(RecursiveDir)\%(FileName)%(Extension)" />
```

I think there is an extra `\` in all of these:
```suggestion
            <Content Include="@(MauiAsset)" Link="%(RecursiveDir)%(FileName)%(Extension)" />
```